test/specs/modules/Dropdown/Dropdown-test.js

```diff
@@ -474,6 +474,28 @@ describe('Dropdown', () => {
 
       instance.setSelectedIndex.should.not.have.been.calledOnce()
     })
+
```

Comment:
Also i notice that you make a call to common.propKeyOnlyToClassName(Dropdown, 'fluid');  Does this need to be done with forceSelection.  I'm not understanding what this class actually test.

Also any reason why the react code isn't in jsx files vs. the js files that i'm seeing

Reply:
> `common.propKeyOnlyToClassName(Dropdown, 'fluid')`

Yes, it tests that `fluid` CSS class is added when prop is defined.

> Also any reason why the react code isn't in jsx files vs. the js files that i'm seeing

It's our code style and we will follow it in future. And yes, I know that there is the [airbnb's rule](https://github.com/airbnb/javascript/tree/master/react#naming) about naming 😄